packages/paste-core/components/file-picker/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {Default, Disabled, Customized} from '../stories/index.stories';
+
+describe('FilePicker', () => {
+  it('should render', () => {
+    const {getByText} = render(<Default data-testid="test-file-picker" />);
+    expect(getByText('Choose a file')).toBeDefined();
+  });
+  it('should render the file input in the DOM', () => {
+    render(<Default data-testid="test-file-picker" />);
+    expect(screen.getByTestId('test-file-picker')).toHaveAttribute('type', 'file');
+  });
+  it('should render as disabled', () => {
+    render(<Disabled data-testid="test-file-picker" />);
+    expect(screen.getByTestId('test-file-picker').getAttribute('aria-disabled')).toBe('true');
+  });
+  it('should set aria-describedby on the file description text', () => {
+    render(<Default data-testid="test-file-picker" />);
+    const textId = screen.getByText('No files selected').getAttribute('id');
+    expect(screen.getByTestId('test-file-picker')).toHaveAttribute('aria-describedby', textId);
+  });
+});
+
+describe('FilePicker customization', () => {
+  it('should set an element data attribute on File Picker', () => {
+    const {container} = render(<Default />);
+    expect(container.querySelector('[data-paste-element="FILEPICKER"]')).toBeInTheDocument();
+    expect(container.querySelector('[data-paste-element="FILEPICKER_BUTTON"]')).toBeInTheDocument();
+    expect(container.querySelector('[data-paste-element="FILEPICKER_TEXT"]')).toBeInTheDocument();
```

Comment:
I know we typically like to have more specific ways of ensuring these get applied correctly, but I couldn't think of any great ones in this case. Lmk if you have thoughts